AudioEncode/encoder.cpp

```diff
@@ -90,7 +90,7 @@ bool aenc_Compress(char *input_filename, char *output_filename, int *input_level
     factor = *input_factor; // Factor of compression (default 4 for 22K, 8 for 44K)
     factor_set = 1;
 
-    if (factor < 1.0f && factor)
+    if (factor > 0.0f && factor < 1.0f)
```

Comment:
this changes logic, is that intended? We don't invert negative factors now

Reply:
There was logic checking that factor is non-zero. Checking for negative value is next after this. Still, I'll change it back to `factor != 0.0f`.